src/components/SubclassCheckbox.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+	<div class="querybuilder__include-subclasses">
+		<input
```

Comment:
Now we need a way to to actually set the checked state from the outside. I.e. some boolean prop (e.g. `value` or `isChecked`) that then is assigned to the `checked` attributed of the input :slightly_smiling_face: .

Reply:
you mean for when it's coming from the store at the start?

Reply:
Yes, exactly

Reply:
gotcha.